acestep/training_v2/ui/flows_train.py

```
@@ -0,0 +1,211 @@
+                )
+            else:
+                print(f"  Preset '{name}' saved ({size} bytes)")
+                print(f"  Location: {path}\n")
+        else:
+            if is_rich_active() and console is not None:
+                console.print(f"  [red]Warning: preset file not found after save: {_esc(path)}[/]\n")
+            else:
+                print(f"  Warning: preset file not found after save: {path}\n")
+    except (KeyboardInterrupt, EOFError):
+        pass
+    except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)